about.html

```diff
@@ -5,16 +5,27 @@
   <div class="col-sm-6">
     <h2>About VuFind®</h2>
 
-    VuFind® is a library resource portal designed and developed for libraries by libraries. The goal of VuFind® is to enable your users to search and browse through all of your library's resources by replacing the traditional OPAC to include:
+    <p>
+      VuFind® is a discovery system designed and developed for libraries by libraries, and also suitable for building
+      search interfaces to all kinds of content. The goal of VuFind® is to enable your users to search and browse
+      through all of your resources in a single consistent and user-friendly interface. Content could include:
+    </p>
     <ul>
-      <li>Catalog Records
-      <li>Digital Library Items
-      <li>Institutional Repository
-      <li>Institutional Bibliography
-      <li>Other Library Collections and Resources
+      <li>Catalog records</li>
+      <li>Institutional repository content</li>
+      <li>Locally hosted journal articles</li>
+      <li>Digitized library materials</li>
+      <li>Items available for interlibrary loan</li>
+      <li>Other collections and resources &mdash; just add metadata!</li>
     </ul>
-
-    VuFind® is completely modular, so you can implement just the basic system or all of the components. Since it's open source, you can modify or add modules to best fit your needs. A wide range of configurable options allow extensive customization without changing any code.
+    <p>
+      VuFind® is completely modular and includes tools to support a wide range of use cases, from a basic Library
+      catalog search to a complex "Bento Box" incorporating data from many sources. It supports building a local
+      index and/or integrating with a variety of existing third-party services. Since it's open source, you can modify or
```

Comment:
Maybe link "open source" to the GitHub organization page?  Maybe also add a sentence about the international nature of the contributions, or the range of organizations, etc.